project/Dependencies.scala

```diff
@@ -127,6 +127,15 @@ object Dependencies {
     )
   )
 
+  val GoogleFcm = Seq(
+    libraryDependencies ++= Seq(
+      "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
+      "com.typesafe.akka" %% "akka-http-spray-json" % AkkaHttpVersion,
+      "com.pauldijou" %% "jwt-core" % "0.14.1",
```

Comment:
For completeness please state the license here.

Reply:
fixed